### DISCUSSION TITLE
How do i expand all the code?

### DISCUSSION THREAD

**Discussion by toplyb** [zed-industries/zed#14533] [Feature Requests]
I can't find the shortcut keys to expand and shrink the code. For example, JetBrains uses command + Shift + - or command + Shift + + to expand and shrink all code. Who can help me?

**Comment by tepavcevic** [comment]
There are key bindings to:

- `fold: Alt + ⌘ + [`
- `unfold: Alt + ⌘ + ]`

and as far as I'm aware you can fold selected code using the `fold` key binding.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
